crates/usvg/src/text/layout.rs

```diff
@@ -2,364 +2,465 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-use std::collections::HashMap;
-use std::num::NonZeroU16;
-use std::sync::Arc;
-
+use crate::tree::{BBox, IsValidLength};
+use crate::{
+    AlignmentBaseline, ApproxZeroUlps, BaselineShift, DominantBaseline, Fill, FillRule, Font,
+    FontStretch, FontStyle, LengthAdjust, PaintOrder, Path, ShapeRendering, Stroke, Text,
+    TextAnchor, TextChunk, TextDecorationStyle, TextFlow, TextPath, TextSpan, Visibility,
+    WritingMode,
+};
 use fontdb::{Database, ID};
 use kurbo::{ParamCurve, ParamCurveArclen, ParamCurveDeriv};
 use rustybuzz::ttf_parser;
+use rustybuzz::ttf_parser::GlyphId;
+use std::collections::HashMap;
```

Comment:
I would prefer the old style: std imports, empty line, external imports, empty line, crate imports.